recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: scotch_6.0.4.tar.gz
+  md5: {{ md5 }}
+  url: http://gforge.inria.fr/frs/download.php/file/34618/scotch_6.0.4.tar.gz
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+ build:
+  - toolchain
+  - zlib 1.2.*  # [unix]
+  - bison
+  - flex
+ run:
+  - zlib 1.2.*  # [unix]
+
+test:
+  commands:
+    # Check for library.
+    - test -f "${PREFIX}/lib/libscotch.a"
+    - test -f "${PREFIX}/lib/libscotcherr.a"
+    - test -f "${PREFIX}/lib/libscotcherrexit.a"
+    - test -f "${PREFIX}/lib/libscotchmetis.a"
+    - test -f "${PREFIX}/lib/libesmumps.a"
```

Comment:
@jakirkham , it looks like the Makefiles are written by hand. Thus, no `./configure` (regarding your question above). Might have to add the flags manually. `-shared` may do the trick. Sorry, if this is already addressed somewhere in the conversations.